posts/mpox-preparedness/index.qmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "2024 mpox outbreak: common analytics tasks and available R tools"
+author:
+  - name: "James Azam"
+    orcid: "0000-0001-5782-7330"
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-147"
+  - name: "Adam Kucharski"
+    orcid: "0000-0001-8814-9421"
+date: "2024-01-02"
+categories: [R, Epiverse-TRACE, mpox, outbreak, outbreak-analytics, DOI]
+bibliography: index.bib
+format:
+  html: 
+    toc: true
+---
+
+There is an ongoing outbreak of mpox globally and countries are on the high alert. The Democratic Republic of Congo is so far the worst hit with a total of 7, 851 cases and 384 deaths between January 1 and May 26, 2024 [^1]. Before 2022, there were no reports of sustained mpox outbreaks in most countries of the world. However, within 1 year (Jan 1, 2022, and Jan 29, 2023), 110 countries in all six WHO Regions had reported a total of 85, 473 confirmed cases and 89 deaths [@laurenson2023description].
+
+Mpox is a highly contagious and severe disease that is transmitted through respiratory droplets and direct contact with infected persons. The disease is characterized by fever, cough, and a rash and has an incubation period between 3 - 34 days, depending on the transmission route [@gessain2022monkeypox]. Infected individuals may experience severe symptoms leading to hospitalisation or death. There are two genetic clades: clade I and clade II, which also has subclades IIa and IIb [@laurenson2023description].
+
+Various analyses of the impact of the outbreak at country level are emerging. The US CDC for example has analysed the potential size of outbreaks resulting from transmission within and between households [^2] and the risk of Clade 1 mpox outbreaks among some key populations associated with key transmission routes [^3]. However, the ongoing outbreak will require a coordinated response from the global health community.
+
+The Epiverse-TRACE team is developing a set of tools to help provide analytics support to decision-makers during this outbreak. This post provides an overview of aspects of analytics that could support decision-making in the context of the ongoing outbreak of mpox.
```

Comment:
```suggestion
The Epiverse-TRACE team is developing a set of analytical tools that could help support to decision-makers during mpox outbreaks. This post provides an overview of the tasks that such tools be applied to.
```
Have edited so it doesn't sound the tools were developed specifically for these outbreaks, and to avoid repetition.